tutor/specs/e2e/my-courses.e2e.ts

```diff
@@ -31,6 +31,15 @@ describe('My Courses', () => {
         expect(
             await page.evaluate(() => document.location.search)
         ).toContain('onboarding=0')
+        await expect(page).not.toHaveSelector('testEl=show-detail', { timeout: 1000 })
+        await page.type('testEl=input-suggested-subject', 'test')
+        await page.click('testEl=offering-0', { force: true })
+        await expect(page).toHaveSelector('testEl=show-detail', { timeout: 1000 })
+        await expect(page).not.toHaveSelector('testEl=submit-suggested-subject', { timeout: 100 })
+        await page.type('testEl=input-suggested-subject', 'test')
+        await expect(page).not.toHaveSelector('testEl=show-detail', { timeout: 100 })
+        await page.click('testEl=offering-0', { force: true })
+        await page.click('testEl=show-detail')
```

Comment:
why the need to `force` the click?  also the `timeout` of 1000  (1 second) is a bit long, I normally use only `100`. Keep in mind that on a `.not.` will always wait that long then test for it to appear at the end of the timeout